.github/workflows/pr-assignment.yml

```diff
@@ -0,0 +1,42 @@
+name: Assign Pull Request to maintainers
+
+on:
+    pull_request:
```

Comment:
Explicitly set `null` value.
```suggestion
    pull_request: ~
```

Reply:
> We should use a [Reusable workflow](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#creating-a-reusable-workflow) instead (stored in our internal repository).

There's a small issue with that:
We will have to store our workflow in a public repo, since https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#allowing-access-to-components-in-an-internal-repository is available only in GitHub Enterprise subscription, which we don't have at the moment.

Reply:
I believe the action template can be stored in a public repo. The contents of the action itself will be public anyway, since we have OS repositories that need this action anyway.

The only vulnerable this is the user assignment data, which can remain in the private repo. We can pass the secret token into the action from leaf repositories, so the secret is not stored in public repo (and potentially abused).

Reply:
@alongosz is reporting an issue, that this doesn't work on forks, but works fine for private repositories.

We might have to take a step back and reconvene.